### ISSUE TITLE
SSL Exclude List #2: Logic

### ISSUE THREAD

**Issue by NghiaTranUIT** [ProxymanApp/proxyman-windows-linux#253]
## Acceptance Criteria
- Make sure it doesn't remove all current SSL Proxying List users from -> It should migrate to the Include List ✅ 
- Wildcard for Exclude List is similar how Wildcard in Include List works
- As soon as we add the Include / Exclude Entry -> Should close connection for this request -> Do not close all alive connection
- Implement the Exclude List, which is similar to IncludeList -> It should have the following logic:
```js
public excludeEntries: SSLEntry[];
private excludeEntriesMap: { [key: string]: number }; // id: index -> For fast-lookup
```

## How to Exclude List Works
- Basically, Exclude List means the list of domains/wildcards should NOT to be intercepted SSL. It works opposite with the Include List.

1. Update this func `shouldIntercept(host: string, port: number): boolean {}`, 
2. Before checking the Include List, we should check the Exclude List first
3. If the entry is matched with Exclude List (Domain / Wildcard) -> Just return `false`
4. If it doesn't match with the Exclude List -> Go to the Include List logic as usual
